presto-parser/src/main/java/io/prestosql/sql/SqlFormatter.java

```diff
@@ -823,7 +823,7 @@ protected Void visitCreateSchema(CreateSchema node, Integer context)
             }
             builder.append(formatName(node.getSchemaName()));
             if (node.getPrincipal().isPresent()) {
-                builder.append(" AUTHORIZATION ")
+                builder.append("\nAUTHORIZATION ")
```

Comment:
separate commit?

Reply:
Let's leave it this was if we can. Matches all the tests now.